dashboard/src/components/UpdateBillingDetailsForm.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+	<FormControl
+		class="mt-4"
+		v-model="billing_name"
+		label="Billing Name"
+	/>
+	<AddressForm
+		ref="address-form"
+		class="mt-4"
+		v-model:address="billingInformation"
+	/>
+	<ErrorMessage
+		class="mt-2"
+		:message="$resources.updateBillingInformation.error"
+	/>
+	<div class="flex w-full justify-end">
+		<Button
+			class="mt-2 w-full sm:w-fit"
+			variant="solid"
+			:loading="$resources.updateBillingInformation.loading"
+			@click="$resources.updateBillingInformation.submit"
+		>
+			Update Billing Details
+		</Button>
+	</div>
+</template>
+
+<script>
+import AddressForm from '@/components/AddressForm.vue';
+import { notify } from '@/utils/toast';
```

Comment:
We are using toast from vue-sonner instead of the old toast. Can you change this?

Reply:
On 2nd look this file seems to be in src folder can you move this to src2?
Having 2 src folders must've confused you. I'll remove the older files soon.

Reply:
okay

Reply:
Moed those pages to src2